packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -96,4 +125,157 @@ class RenderAndroidView extends RenderBox {
+
+    dispatchPointerEvent(event);
+
+    if (event is PointerUpEvent) {
+      pointerPositions.remove(event.pointer);
+      pointerProperties.remove(event.pointer);
+      if (pointerProperties.isEmpty) {
+        nextPointerId = 0;
+        downTimeMillis = null;
+      }
+    }
+    if (event is PointerCancelEvent) {
+      pointerPositions.clear();
+      pointerProperties.clear();
+      nextPointerId = 0;
+      downTimeMillis = null;
+    }
+  }
+
+  void dispatchPointerEvent(PointerEvent event) {
+    final List<int> pointers = pointerPositions.keys.toList();
+    final int pointerIdx = pointers.indexOf(event.pointer);
+    final int numPointers = pointers.length;
+
+    // Android MotionEvent objects can batch information on multiple pointers.
+    // Flutter breaks these such batched events into multiple PointerEvent objects.
+    // When there are multiple active pointers we accumulate the information for all pointers
+    // as we get PointerEvents, and only send it to the embedded Android view when
+    // we see the last pointer. This way we achieve the same batching as Android.
+    if(isSinglePointerAction(event) && pointerIdx < numPointers - 1)
+      return;
+
+    int action;
+    switch(event.runtimeType){
+      case PointerDownEvent:
+        action = numPointers == 1 ? AndroidViewController.kActionDown
+            : AndroidViewController.pointerAction(pointerIdx, AndroidViewController.kActionPointerDown);
+        break;
+      case PointerUpEvent:
+        action = numPointers == 1 ? AndroidViewController.kActionUp
+            : AndroidViewController.pointerAction(pointerIdx, AndroidViewController.kActionPointerUp);
+        break;
+      case PointerMoveEvent:
+        action = AndroidViewController.kActionMove;
+        break;
+      case PointerCancelEvent:
+        action = AndroidViewController.kActionCancel;
+        break;
+      default:
+        return;
+    }
+
+    final AndroidMotionEvent androidMotionEvent = new AndroidMotionEvent(
+        downTime: downTimeMillis,
+        eventTime: event.timeStamp.inMilliseconds,
+        action: action,
+        pointerCount: pointerPositions.length,
+        pointerProperties: pointers.map((int i) => pointerProperties[i]).toList(),
+        pointerCoords: pointers.map((int i) => pointerPositions[i]).toList(),
+        metaState: 0,
+        buttonState: 0,
+        xPrecision: 1.0,
+        yPrecision: 1.0,
+        deviceId: 0,
+        edgeFlags: 0,
+        source: 0,
+        flags: 0
+    );
+    viewController.sendMotionEvent(androidMotionEvent);
+  }
+
+  AndroidPointerCoords coordsFor(PointerEvent event) {
+    final Offset position = globalToLocal(event.position);
+    return new AndroidPointerCoords(
+        orientation: event.orientation,
+        pressure: event.pressure,
+        // Currently the engine omits the pointer size, for now I'm fixing this to 0.33 which is roughly
+        // what I typically see on Android.
```

Comment:
Is this screen density dependent?

Reply:
Documentation for the field is kind of vague...

"A normalized value that describes the approximate size of the pointer touch area in relation to the maximum detectable size of the device."

I'm guessing device is the pointing device.